crucible-llvm/src/Lang/Crucible/LLVM/SymIO.hs

```diff
@@ -557,7 +557,7 @@ callWriteFileHandle memOps fsVars filedesc buf count =
 symio_overrides
   :: (IsSymInterface sym, HasLLVMAnn sym, HasPtrWidth wptr, wptr ~ ArchWidth arch, ?memOpts :: MemOptions)
   => LLVMFileSystem wptr
-  -> [OverrideTemplate p sym arch rtp l a]
+  -> [OverrideTemplate p sym LLVM arch]
```

Comment:
Any reason not to make this polymorphic in the `ext` type parameter, similar to the other LLVM overrides?